### PR TITLE
Fix mobile perf by disabling scroll animation.

### PR DESCRIPTION
**Pull request by anselmh**
@drublic please review :)

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -199,6 +199,7 @@
 	 */
 	(function () {
 		var $window = $(window);
+		var mq = window.matchMedia( "(min-width: 672px)" );
```
Comment (drublic) [inline main.js]: Maybe check for window.matchMedia:

```
window.matchMedia && window.matchMedia('(min-width: 672px)');
```

**Comment by drublic** [comment]
:+1:

**Comment by anselmh** [comment]
added test

**Comment by drublic** [comment]
Gut.

**Pull request merged**